src/weather_man.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+const rootUrl = 'http://api.openweathermap.org/data/2.5/weather?q=';
+const APIKey = '&APPID=81ebd306a97bf6e5342257562b958514';
+
+class WeatherMan extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {city: "",
+                  weather: ""
+                 }
+  }
+
+  componentWillMount() {
+    const url = rootUrl + this.props.city + APIKey;
```

Comment:
Maybe this be more expressive if the call to the api to fetch data was  a separate function like getCityData(city)?